From e418325866ac4e90d86dcd83accacdd905b1a307 Mon Sep 17 00:00:00 2001
From: mudler <mudler@sabayon.org>
Date: Fri, 16 Sep 2016 12:41:35 +0200
Subject: [PATCH] Revert "Revert "remove
 app-emulation/virtualbox-guest-additions if not in vbox after install""

This reverts commit 054a714e4b209eca6c560b05cca1cef04c309ecb.
---
 pyanaconda/sabayon/utils.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/pyanaconda/sabayon/utils.py b/pyanaconda/sabayon/utils.py
index b493dcc..1dfc631 100644
--- a/pyanaconda/sabayon/utils.py
+++ b/pyanaconda/sabayon/utils.py
@@ -670,6 +670,9 @@ blacklist nouveau
         try:
             repo = self._backend.entropy.installed_repository()
 
+            if not self.is_virtualbox:
+                self.remove_package("virtualbox-guest-additions")
+
             for package in packages:
 
                 pkg_id, _pkg_rc = repo.atomMatch(package)
-- 
2.9.2

